InsertGenerator/main.py

```diff
@@ -6,7 +6,7 @@
 
 
 def main():
-    cfg_helper = ConfigHelper()
+    cfg_helper = ConfigHelper(path='configs/config.example.ini')
```

Comment:
config.example.ini is not supposed to store actual values and be used at all. It is not ignored by GIT and allows us to show everyone what our config file looks like. When you clone this repository, you can just clone this file, rename it to config.ini and set your real connections and other sensitive data. It's an example of our config file.